### ISSUE TITLE
Can you provide the features extracted by resnet151？thank you very much

### ISSUE THREAD

**Issue by aiwolflow666** [FloretCat/CMRAN#2]
I cannot use the resnet151 features I extracted myself to reproduce the results in the paper

**Comment by FloretCat** [comment]
Hi,
Thanks for your interest in our work.

For the ResNet151 features, we only sample 8 frames for each segment in a video and it may be better to sample 16 frames. We provide the features we used on Baidu Cloud [here](https://pan.baidu.com/s/1aX_ix95xqeaPxjdXmedDxA )(pwd: fa0e) (sorry to fail to upload it to google drive as the space limitation). Also, to reproduce the results, you may need to adjust the learning schedule, such as 
`scheduler = MultiStepLR(optimizer, milestones=[15, 40, 60], gamma=0.5)`.
After downloading the features, you can read the file using
 `visual_feats = np.load('PATH/TO/resnet_features.npy', mmap_mode='r')`.
In the numpy file, the visual features are organized in the same order as the h5 file of VGG features.

Regards

**Comment by aiwolflow666** [comment]
Thank you for the features, but I’m sorry, I can’t use the resnet151 features you provided to reproduce the results in your paper, and can only achieve an accuracy of 76.592%. The learning rate and batch size are still consistent with vgg19, and the learning rate is the same. The scheduler method is replaced with the latest one you provide

**Comment by FloretCat** [comment]
Hi，
Due to random factors, you can try a few more seeds. A performance difference of about 1% is normal.

**Comment by aiwolflow666** [comment]
I used the seed number provided in your code for randomization, but the results in your paper cannot be obtained. Theoretically, the seed number is fixed and the error will not be large.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月15日(星期四) 下午2:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [FloretCat/CMRAN] Can you provide the features extracted by resnet151？thank you very much (#2)





 
Hi，
 Due to random factors, you can try a few more seeds. A performance difference of about 1% is normal.

**Comment by FloretCat** [comment]
We re-ran our raw code for resnet features and achieved a comparable performance (i.e, 78.83 in the supervised setting and 75.25 in the weakly supervised setting). We provide the code and trained models [here](https://pan.baidu.com/s/1zs6nl2EoqNlPVZwoVHn4aA) (pwd: a3nj) and hope these will be helpful for you. It is worth noting that our method may affect more easily by random factors when using resnet features, and it is normal to have a performance difference of around 1%.

**Comment by aiwolflow666** [comment]
Thank you very much for your reply, I will test it




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月17日(星期六) 中午1:36
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [FloretCat/CMRAN] Can you provide the features extracted by resnet151？thank you very much (#2)





 
We re-ran our raw code for resnet features and achieved a comparable performance (i.e, 78.83 in the supervised setting and 75.25 in the weakly supervised setting). We provide the code and trained models here (pwd: a3nj) and hope these will be helpful for you. It is worth noting that our method may affect more easily by random factors when using resnet features, and it is normal to have a performance difference of around 1%.
